client/process.go

```diff
@@ -14,12 +14,14 @@ type ProcessClient commonClient
 type ProcessListOptions struct {
 	*ListOptions
 
-	GUIDs             Filter `qs:"guids"`
-	Types             Filter `qs:"types"`
-	Names             Filter `qs:"names"`
```

Comment:
Drop `names` as this is [no query parameter](http://v3-apidocs.cloudfoundry.org/version/3.165.0/index.html#list-processes)